docs/SPECS.md

```diff
@@ -138,26 +141,27 @@ hash of the group file is used during a resharing procedure to make sure node
 are resharing from the correct group.
 The hash is derived using the blake2b hash function.  The Go procedure works as
 follow:
+
 ```go
 func (g *Group) Hash() []byte {
     h, _ := blake2b.New256(nil)
     // sort all nodes entries by their index
-	sort.Slice(nodes, func(i, j int) bool {
-		return nodes[i].Index < nodes[j].Index
-	})
-	// all nodes public keys and positions
-	for _, n := range nodes {
-		h.Write(n.Hash())
-	}
-	binary.Write(h, binary.LittleEndian, uint32(g.Threshold))
-	binary.Write(h, binary.LittleEndian, uint64(g.GenesisTime))
-	if g.TransitionTime != 0 {
-		binary.Write(h, binary.LittleEndian, g.TransitionTime)
-	}
-	if g.PublicKey != nil {
-		h.Write(g.PublicKey.Hash())
-	}
-	return h.Sum(nil)
+    sort.Slice(nodes, func(i, j int) bool {
+        return nodes[i].Index < nodes[j].Index
+    })
+    // all nodes public keys and positions
+    for _, n := range nodes {
+        h.Write(n.Hash())
+    }
+    binary.Write(h, binary.LittleEndian, uint32(g.Threshold))
+    binary.Write(h, binary.LittleEndian, uint64(g.GenesisTime))
+    if g.TransitionTime != 0 {
+    binary.Write(h, binary.LittleEndian, g.TransitionTime)
+    }
+    if g.PublicKey != nil {
+    h.Write(g.PublicKey.Hash())
+    }
+    return h.Sum(nil)
```

Comment:
golang uses tabs not spaces - please can it be this way around so that folks don't get linter errors when copy/pasting this?

Reply:
Rolled back. :)